weave/trace_server/orm.py

```diff
@@ -171,7 +177,11 @@ def truncate_sql(self, database_type: DatabaseType) -> str:
         if database_type == "sqlite":
             return f"DELETE FROM {self.name}"
 
-    def tuple_to_row(self, tup: typing.Tuple, fields: list[str]) -> Row:
+    def tuple_to_row(
+        self,
+        tup: typing.Union[typing.Tuple, typing.Sequence[typing.Any]],
```

Comment:
why these `typing.Sequence[typing.Any]`?

Reply:
more precise type. mypy is flagging a (valid) type error here.